### PR TITLE
CDAP-7829 Fix a explore failure on Windows

### PR DESCRIPTION
**Pull request by anwar6953**
Cherry-pick of https://github.com/caskdata/cdap/pull/7363, to release/3.5.

http://builds.cask.co/browse/CDAP-RUT482-4

### CODE REVIEW

**Comment by CuriousVini** [comment]
lgtm

File: cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java [hunk 1]
```diff
@@ -174,7 +174,7 @@ private static File createLocalDir(CConfiguration cConf, String dirName) {
       File credentialsDir = new File(localDirStr, dirName);
 
       try {
-        java.nio.file.Files.createDirectories(credentialsDir.toPath(), FileUtils.OWNER_ONLY_RWX);
```
Comment (anew) [inline cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java:177]: There must be a reason this used to be created with that set of permissions. What breaks if we remove that?

Comment (anwar6953) [reply cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java:177]: The actual credential files written inside this directory are written with restrictive permissions, so the restrictive permissions on the directory aren't necessary in the same manner.

Comment (anew) [reply cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java:177]: ok. LGTM

**Review by anew — APPROVED** [review]

**Pull request merged**